backend/command/src/shared/validators/is-bigger-than.ts

```diff
@@ -0,0 +1,33 @@
+// Full credits to:
+// https://stackoverflow.com/questions/59980341/can-i-compare-number-variables-with-class-validator/67815042#67815042
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+} from 'class-validator';
+
+export function IsBiggerThan(
+  property: string,
+  validationOptions?: ValidationOptions,
+) {
+  return function (object: unknown, propertyName: string) {
+    registerDecorator({
+      name: 'isBiggerThan',
+      target: object.constructor,
+      propertyName: propertyName,
+      constraints: [property],
+      options: validationOptions,
+      validator: {
+        validate(value: any, args: ValidationArguments) {
```

Comment:
does value have to be any? Wouldn`t number also work?

Reply:
We do not know the type of the "other" field we want to compare with. It could be a string as well or anything else.